libraries/grpc-sdk/src/utilities/registerMigrations.ts

```diff
@@ -0,0 +1,32 @@
+import { readFileSync, readdirSync } from 'fs';
+import { DatabaseProvider } from '../modules';
+
+/**
+ * Registers all the migrations of a module and then triggers them
+ * There could be many migration files or one that contains all migrations
+ * @param database
+ * @param moduleName
+ * @param moduleVersion => version of the module that registers migrations
+ * @param path => absolute path to module's migration folder
+ **/
+export async function registerMigrations(
+  database: DatabaseProvider,
+  moduleName: string,
+  moduleVersion: string,
+  path: string,
+) {
+  const files = readdirSync(path).filter(f => f.endsWith('.js'));
+  const migrations = new Map<string, string>();
+  for (const name of files) {
+    // Ignore authentication's config migration for convict
+    if (name === 'configMigration.js' && moduleName === 'authentication') continue;
```

Comment:
instead of adding a specific file here, you should check if the file includes the up and down properties for example.